md2conf/__main__.py

```diff
@@ -81,6 +81,26 @@ def main() -> None:
         const=None,
         help="Do not add 'generated by a tool' prompt to pages.",
     )
+    parser.add_argument(
+        "--render-mermaid",
+        dest="render_mermaid",
+        action="store_true",
+        default=True,
+        help="Render Mermaid diagrams as image files and add as attachments.",
+    )
+    parser.add_argument(
+        "--no-render-mermaid",
+        dest="render_mermaid",
+        action="store_false",
+        help="Inline mermaid diagram in the confluence page.",
+    )
+    parser.add_argument(
+        "--render-mermaid-format",
+        dest="kroki_output_format",
+        choices=["png", "svg"],
+        default="png",
+        help="Format for rendering mermaid diagrams (default: 'png').",
```

Comment:
Did you get Confluence work with SVG? I have had trouble embedding SVG in wiki pages, which is why *md2conf* is currently referencing a PNG image instead of an SVG image when both formats are found.